cli/tools/installer.rs

```diff
@@ -139,6 +139,47 @@ pub fn infer_name_from_url(url: &Url) -> Option<String> {
   Some(stem)
 }
 
+pub fn uninstall(name: String, root: Option<PathBuf>) -> Result<(), AnyError> {
+  let root = if let Some(root) = root {
+    canonicalize_path(&root)?
+  } else {
+    get_installer_root()?
+  };
+  let installation_dir = root.join("bin");
+
+  // ensure directory exists
+  if let Ok(metadata) = fs::metadata(&installation_dir) {
+    if !metadata.is_dir() {
+      return Err(generic_error("Installation path is not a directory"));
+    }
+  }
+
+  let mut file_path = installation_dir.join(&name);
+
+  let mut removed = false;
+
+  if file_path.exists() {
+    fs::remove_file(&file_path)?;
+    println!("deleted {}", file_path.to_string_lossy());
+    removed = true
+  };
+
+  if cfg!(windows) {
+    file_path = file_path.with_extension("cmd");
+    if file_path.exists() {
+      fs::remove_file(&file_path)?;
+      println!("deleted {}", file_path.to_string_lossy());
+      removed = true
+    }
+  }
+
+  if !removed {
+    return Err(generic_error(format!("No installation found for {}", name)));
+  }
+  println!("✅ Successfully uninstalled {}", name);
+  Ok(())
+}
+
 pub fn install(
```

Comment:
In the install command there are some `extra_files`. I think those need to be checked for and removed in the uninstall command.

Reply:
I have updated to clean the extra files.